aziotctl/aziotctl-common/src/restart.rs

```diff
@@ -0,0 +1,55 @@
+use std::io::{self, Write};
+use std::process::Command;
+
+use anyhow::{Context, Result};
+
+use crate::ServiceDefinition;
+
+#[allow(clippy::missing_errors_doc)]
```

Comment:
https://github.com/Azure/iot-identity-service/blob/main/CONTRIBUTING.md

See 3.vi

ditto for the other instances